wiki/Help_Centre/bg.md

```diff
@@ -0,0 +1,79 @@
+| [Бийтмапове и редактор](/wiki/Help_Centre/Beatmapping) | Поддържане на бийтмапове, собственост на бийтмапове, слотове за бийтмапове |
+| [Клиент за игра](/wiki/Help_Centre/Client) | Бъгове и грешки, игра, свързване, производителност |
+| [Инсталиране и регистриране](/wiki/Help_Centre/Installation_and_registration) | Изтегляне на играта, създаване на профил |
+| [Магазин](/wiki/Help_Centre/Store) | Стоки, osu!go, osu!keyboard, osu!tablet |
+| [Уебсайт](/wiki/Help_Centre/Website) | Блокиране на потребител, връзка с поддръжката, изглед на сайта |
+
+## Помогнете ни да ви помогнем
+
+### Лог файлове
+
+**Файловете с логове са детайлни записки на клиента за игра относно неговата работа във всеки един момент. Можем да ги използваме за да разберем какво причинява проблема ви.**
+
+Тези файлове са изключително полезни и могат да направят разрешаването на сложни проблеми нищо работа.
+
+Ако някой от поддръжката поиска такива файлове, ето как да ги предоставите:
+
+1. Отворете osu!.
+2. Кликнете върху бутона `Options` в главното меню или натиснете `Ctrl` + `O`.
+3. Напишете `release` в полето за търсене. Това ще ви отведе към текущата версия която клиентът за игра използва.
+4. Уверете се че е избран `Cutting Edge (Експериментален)`.
+5. Кликнете върху бутона за рестартиране, който ще се появи в долната част на екрана, в случай че сте направили промяна във версията на клиента, за да потвърдите промените.
+6. Отидете отново в `Options` и кликнете върху `Open osu! folder`.
+7. Отворете папката `Logs` от прозореца който ще се отвори.
+8. Изберете необходимия лог файл (представителя на поддръжката ще ви каже кой) и го прикачете в писмото към поддръжка или публикацията във форум.
+
+### Event Viewer
+
+**Event Viewer е вградена програма към операционната система Windows, която може да се използва за откриване на логове с грешки когато osu! отказва да създаде. Тези логове с грешки могат да бъдат полезни за да се разбере причинителя на проблема.**
+
+При определени грешки, когато osu! спре да работи и не ви дава логове за грешки, единственото място на което може да откриете такива е Event Viewer.
+
+Ако някой от поддръжката поиска логове за грешки чрез Event Viewer, ето как да ги предоставите:
+
+1. Когато osu! спре да работи, натиснете `Win` + `R` за изпълнение на команда.
+2. В прозореца за изпълнение напишете `eventvwr` и натиснете `Enter`. това ще отвори Event Viewer.
+3. В прозореца на Event Viewer, в ляво, кликнете `Windows Logs` след това `Application`.
+4. В дясната част, кликнете `Filter current log`.
+5. В прозореца който ще се отвори, уверете се че сте отбелязали с отметка `Error` и кликнете `OK`.
+6. Натиснете `Ctrl` + `F` (или кликнете `Find...` от дясно) и напишете osu! в прозореца за търсене. Това ще намери лога за грешка на osu!.
+7. Отидете в раздела `Details` от долната част, разширете `System` и `Event Data` с кликване върху тях.
+8. Копирайте текста от там и го поставете в писмото към поддръжка или публикацията във форум.
```

Comment:
by any chance, if you have Windows in Bulgarian, could you use translated interface labels here as well instead of "Windows Logs" etc.?

Reply:
Unfortunately, Microsoft never translated the event viewer. Im with win 11 right now and is still the same.

Reply:
then we're all set